### ISSUE TITLE
Oxygen Vent / CO2 Scrubber inconsistent

### ISSUE THREAD

**Issue by evacfk** [zmaster587/AdvancedRocketry#1146]
Minecraft Version: 1.12.2
-
Forge Version: 14.23.2.2613
-
Advanced Rocketry Version:  AdvancedRocketry-1.12.2-1.2.6-54
-
LibVulpes Version: LibVulpes-1.12.2-0.2.8-27-universal
-
**is this the latest curseforge build?**

Version info:  http://evac.gg/i/x5ymt.png

Explain issue in detail:

The oxygen vent and co2 scrubbers are very inconsistent, in some rooms they work fine. Other times they do not.

In this room it does not work, co2 scrubbers do not light up and taking off suit causes choking
http://evac.gg/i/7x3t3.png

In the test box I created it works fine: http://evac.gg/i/89u5f.png

Main room it sometimes works: http://evac.gg/i/0vqxk.png

These are setup with a power source, lever for redstone signal, scrubbers on each side and cartridges loaded into each scrubber.

**Comment by DarVillous** [comment]
http://evac.gg/i/89u5f.png shows you are in creative.  are you sure it is working?
Consider adding an atmosphere analyzer to your helmet so you can see the atmosphere type and if it is breathable.  

I have never placed my vents or scrubber with "open space" touching them.  (meaning I always have a block separating them from the vacuum.  because the vent (more or less) fills a 16 block radius from the vent mine tend to be in the middle of a room instead of on (or in) the outer wall.

I also have never put a lever on the front of the vent because I wasn't sure if it would block the oxygen flow.  I just toggle the redstone mode in the gui.

**Comment by evacfk** [comment]
I'm in creative because I had my suit off while testing the oxygen vent.
I test the environment by swapping in/out of creative.

Is the block separating them crucial? I'm powering this via solar panels so it goes [Vacuum][Solar Panel][Oxygen vent]

http://evac.gg/i/89u5f.png - lever is in front of the vent and it works fine.

**Comment by zmaster587** [comment]
If you use the tracer when it's not working, can to determine where it's leaking?

**Comment by kyle-sama** [comment]
Playing in the latest ATM3 modpack on a server, and I'm having similar problems. First off, the oxygen tracer never seems to work. 
When I built a space station the first time, my first room I was able to oxygenize. Afterwards, I can't seem to get any other room to fill with oxygen with various setups. Also, after turning off the oxygen vent in the initial room, it still has oxygen in it, even when I open an airlock, that room doesn't lose oxygen.
